pymc3/tests/test_distributions.py

```diff
@@ -1907,6 +1906,8 @@ def test_matrixnormal(self, n):
                 "colcov": PdMatrix(n) * mat_scale,
             },
             matrix_normal_logpdf_cov,
+            extra_args={"size": n},
+            decimal=select_by_precision(float64=5, float32=5),
```

Comment:
Do the tests pass if aesara `floatX` is set to "float32"? Usually the default precision we use for float32 is `3`. Your change is increasing the precision compared to what we were testing before.

Reply:
Also make sure you test all combinations locally by setting `n_samples=-1`

Reply:
I set the precision to 2 decimals for `float32` just to be sure.

Reply:
You should actually test. 2 might not be necessary or it might not be enough.

Reply:
You can check the notes 3 and 4 in here: https://github.com/pymc-devs/pymc3/blob/fe115b857ae42e84d1f359153fd1f7efb2c327ae/docs/source/distribution_implementation.md#4-adding-tests-for-the-logp--logcdf-methods

Reply:
You were right, the passed all the way upto 4 decimal points in `float32` mode. And started showing failures at 5 decimal points. I left it at our default of 3 cause it may show flaky failures at 4 decimal points.